test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -61,6 +62,14 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
     }
+    
+    public void emptyArrayList() throws Exception {
```

Comment:
You have not the naming conventions for test methods. Note the naming convention for test methods (the first part is feature/method under test), then the test scenario you are testing, then the expected behaviour.
E.g. isAnyNull_emptyList_falseReturned
Part 2 and 3 of the test name can be removed to indicate an increased test scope

You have not annotated your test methods as @Test